templates/layout/page.html.twig

```diff
@@ -10,21 +10,19 @@
 <header
   class="ecl-site-header-{{ ecl_branding }}"
   data-ecl-auto-init="SiteHeader{{ ecl_branding|capitalize }}"
+  {{ ecl_has_menu_attribute }}
```

Comment:
We need to test this is added properly.

Reply:
I've updated behat tests for this purpose.